### DISCUSSION TITLE
aspect-ratio is not working on target 99 (gift-box)

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#201] [Q&A]
It looks as if it works, but it doesn't. I get only 95.9% match
Can me anyone tell why?

**Comment by programus** [comment]
`aspect-ratio` works for me in other targets I used it. Maybe there are some other problems. I think paste your code here would let others help you easier.

**Reply by verberg** [reply]
Yes, I use aspect-ratio in other targets and it works.
Here ist the code:
<p><style>*{width:60;height:60;margin:110 130;background:#AC474B;color:#fff;box-shadow:inset 9in 0,0 85q}*>*{margin:0 80}p{position:fixed;width:40;height:40;margin:-60 -45;border-radius:50%50%0;box-shadow:inset 0 0 0 10q;-webkit-box-reflect:right -10px

**Reply by programus** [reply]
I can’t find `aspect-ratio` in your code…

**Reply by verberg** [reply]
Oh, sorry, I took the code, that gets 100% match.
And now, it's crazy, it dosn't look as 100% (but on 24.12 it did)

<p><style>*{width:60;aspect-ratio:1;margin:110 130;background:#AC474B;color:#fff;box-shadow:inset 9in 0,0 85q}*>*{margin:0 80}p{position:fixed;width:40;margin:-60 -45;border-radius:50%50%0;box-shadow:inset 0 0 0 10q;-webkit-box-reflect:right -10px

**Reply by verberg** [reply]
And I do'nt know why: this code gives 100% matsch WITH aspect-ratio (crazy!!!)
<p><style>*{width:60;aspect-ratio:1;margin:110 130 130;background:#AC474B;color:#fff;box-shadow:inset 9in 0,0 85q}*>*{margin:0 80}p{position:fixed;width:40;margin:-60 -45;border-radius:50%50%0;box-shadow:inset 0 0 0 10q;-webkit-box-reflect:right -10px

**Reply by programus** [reply]
I think they are keeping updating it. I had a similar problem ( https://github.com/cssbattle/forum/discussions/203 ) and in the answer, you can see they are going to fix it in the future updates.

**Reply by verberg** [reply]
oh, i missed a better rank? ;)

**Reply by programus** [reply]
No. `height:60` is shorter than `aspect-ratio:1`, so you should have a better rank if you don't use `aspect-ratio`.

**Reply by programus** [reply]
@verberg Sorry, I am afraid I cannot understand your code well. Do you mind telling my why there are 4 squares for only one `body` tag with 2 `box-shadow`?
